recipes/zeromq/all/conandata.yml

```diff
@@ -9,14 +12,34 @@ sources:
     url: "https://github.com/zeromq/libzmq/releases/download/v4.3.2/zeromq-4.3.2.tar.gz"
     sha256: "ebd7b5c830d6428956b67a0454a7f8cbed1de74b3b01e5c33c5378e22740f763"
 patches:
+  "4.3.5":
+    - patch_file: "patches/0004-cmake-minimum-required-first-4.3.5.patch"
+      patch_description: "exchange positions between project and cmake_minimum_required"
+      patch_type: "portability"
   "4.3.4":
     - patch_file: "patches/0003-rpath-macos-4.3.3.patch"
+      patch_description: "disable rpath on macOS"
+      patch_type: "portability"
     - patch_file: "patches/0004-cmake-minimum-required-first.patch"
+      patch_description: "exchange positions between project and cmake_minimum_required"
+      patch_type: "portability"
   "4.3.3":
     - patch_file: "patches/0003-rpath-macos-4.3.3.patch"
+      patch_description: "disable rpath on macOS"
```

Comment:
What do you mean? This patch doesn't disable `rpath`.

Reply:
@SpaceIm
Thank you for youre comment.
Now I understand.